### DISCUSSION TITLE
Look at the series, determine the pattern, and find the value of the unknown number!

### DISCUSSION THREAD

**Discussion by NomanGhiasy** [ParanoidUser/yolo#1110] [Galaxy Trivia] [answered]
0, 1, 3, 6, 10, 15, x

**Comment by ParanoidUser** [comment]
In could be Blackjack🃏 or **21**, as it follows the order *Prev N + Seq N = Next N* producing:
```
N0  + #1 = N1
N1  + #2 = N3
N3  + #3 = N6
N6  + #4 = N10
N10 + #5 = N15
N15 + #6 = N21
```

**Comment by sheri-na** [comment, marked as answer]
The pattern is 0+1 =1 , 1+2 = 3, 3+3 = 6, 6+4 = 10 , 10 + 5 = 15, 15 +6 = 21
so the next number is 21

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/minamin33?achievement=galaxy-brain&tab=achievements) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉
